vocabularies/Common.xml

```diff
@@ -945,6 +945,10 @@ The minimum structure is
           <Annotation Term="Core.Description" String="Action that creates a new draft" />
           <Annotation Term="Core.LongDescription" String="New drafts may also be created by POSTing an empty entity without any properties to the entity set." />
         </Property>
+        <Property Name="AdditionalNewActions" Type="Collection(Common.QualifiedName)" Nullable="true">

```

Comment:
```suggestion
        <Property Name="AdditionalNewActions" Type="Collection(Common.QualifiedName)" Nullable="false">
```
For collections the `Nullable` means the items, and we don't want null values in the collection.